CONTRIBUTING.md

```diff
@@ -118,10 +118,10 @@ This section shows you how to send a pull request. The main staff are always mon
 
 This section shows you how to write commit messages in Clay. Follow these guidelines to help us maintain order and make it easier to locate your changes.
 
-Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope and a subject:
+Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope, a jira ticket number and a subject:
 
 ```
-<type>(<scope>): <subject>
+<type>(<scope>): <jira ticket number> - <subject>
```

Comment:
I have a preference for keeping the commit message cleaner... so I would just leave a blank space between the ticket number and the subject instead of the `-`. What do you think?

Reply:
Sounds good! 
Thanks @matuzalemsteles

Reply:
I like Matu's idea too. This is the way I have been writing it.